internal/env/env.go

```diff
@@ -0,0 +1,46 @@
+package env
+
+import (
+	env "github.com/caarlos0/env/v10"
+	"github.com/joho/godotenv"
+	log "github.com/sirupsen/logrus"
+)
+
+type Env struct {
+	ServerPort        int				`env:"SERVER_PORT" envDefault:"8081"`
+	K1AccessToken     string	  `env:"K1_ACCESS_TOKEN"`
```

Comment:
could you align this, are you using the golang extension or format on save vscode config?

Reply:
this is something that is just showing up oddly. locally presents how it should 🤷

Reply:
your're right, however in local your branch looks:
![CleanShot 2023-11-10 at 19 37 08](https://github.com/kubefirst/kubefirst-api/assets/3932407/dfd3d008-034a-4d48-afaa-31efc68c21b9)

Reply:
![](https://media.giphy.com/media/ukGm72ZLZvYfS/giphy.gif)

<img width="755" alt="Screenshot 2023-11-10 at 4 41 06 PM" src="https://github.com/kubefirst/kubefirst-api/assets/76881293/d6263971-da65-40ae-870a-38cebab3c2c7">